__tests__/app.test.js

```diff
@@ -115,6 +115,74 @@ describe('/api/reviews/:review_id', () => {
                 });
         });
     });
+    describe('PATCH', () => {
+        test('should update the votes of a review (adding)', () => {
+            const updateVotes = {inc_votes: 50}
+            return request(app)
+            .patch("/api/reviews/1")
+            .send(updateVotes)
+            .expect(200)
+            .then(({body}) => {
+                expect(body.review).toEqual({"category": "euro game",
```

Comment:
Would avoid `.toEqual` here for the same reason as before - but still assert specific values within as you have